code/modules/mob/living/simple_animal/hostile/ordeal/steel.dm

```diff
@@ -200,17 +216,16 @@
 	deathmessage = "mutters something under their breath before collapsing."
 	mob_biotypes = MOB_ORGANIC|MOB_HUMANOID|MOB_BUG
 	ranged_cooldown_time = 15 SECONDS
-	a_intent = INTENT_HELP
+	move_to_delay = 3
 	maxHealth = 2000
 	health = 2000
-	melee_damage_lower = 40
-	melee_damage_upper = 57
+	melee_damage_lower = 50
+	melee_damage_upper = 55
 	vision_range = 12
 	move_to_delay = 3
 	ranged = TRUE
-	retreat_distance = 2
```

Comment:
Doesn't the removal of this mean they won't backup to use their AoE?

Reply:
Retreating makes them way too weak, as people would just run them into a corner, forcing them to do literally nothing and wait for death.

Reply:
Thats fine.